### DISCUSSION TITLE
Unexpected IG Logout

### DISCUSSION THREAD

**Discussion by brxndonnn** [new92/igfi#29] [Q&A] [answered]
why does my IG logout when i run the script, is it because i use it to much or what?

I have to say that IG notified me that my acc had bot behavior so i stopped using the script for a few days but when i recently ran it again my IG logs out on my phone and PC and then i have to log in again

any idea on why that may be?

**Comment by new92** [comment, marked as answer]
Hey there ! So the problem seems to come from the module which the script uses (instaloader). More specifically, instaloader uses a virtual machine to log into your account so Instagram doesn't recognise the device and marks it as a bot. So the more often you use the script the more Instagram will mark it as a bot activity. So what I suggest is to use IGFollowersIncreaser every 3 to 4 days. Hope this helped ! If you are facing any other problems don't hesitate to contact me !
